runtime/compiler/control/JITServerCompilationThread.cpp

```diff
@@ -82,6 +87,52 @@ outOfProcessCompilationEnd(
 
    // Pack log file to send to client
    std::string logFileStr = TR::Options::packLogFile(comp->getOutFile());
+   std::cout << "Pre Persistent Logging Section" << std::endl;
+   std::cout << comp->getOption(TR_PersistLogging) << std::endl;
+#ifdef PERSISTENT_LOGGING_SUPPORT
+
+   std::cout << "Post Persistent Logging Section" << std::endl;
+   std::cout << comp->getOption(TR_PersistLogging) << std::endl;
+//   if (comp->getOption(TR_PersistLogging)) { TODO: Uncomment out and fix this flag
+      uint64_t clientUID = entry->getClientUID();
+      const char* methodSignature = compInfoPT->getCompilation()->signature();
+
+      std::cout << "Persistent Logging enabled" << std::endl;
+      std::cout << "Found Client id: " << clientUID << std:: endl;
+      printf("potential method full name: %s\n",methodSignature);
+      uint32_t persistentLoggingDatabasePort = compInfoPT->getCompilationInfo()->getPersistentInfo()->getJITServerPersistentLoggingDatabasePort();
+      std::cout << "what is the persistent logging database port ? " << persistentLoggingDatabasePort << std::endl;
+      std::string persistentLoggingDatabaseAddress = compInfoPT->getCompilationInfo()->getPersistentInfo()->getJITServerPersistentLoggingDatabaseAddress();
+      std::cout << "what is the persistent logging database Address ? " << persistentLoggingDatabaseAddress << std::endl;
+
+      std::string persistentLoggingDatabaseUsername = compInfoPT->getCompilationInfo()->getPersistentInfo()->getJITServerPersistentLoggingDatabaseUsername();
+      std::cout << "what is the persistent logging database Username ? " << persistentLoggingDatabaseUsername << std::endl;
+
+      std::string persistentLoggingDatabasePassword = compInfoPT->getCompilationInfo()->getPersistentInfo()->getJITServerPersistentLoggingDatabasePassword();
+      std::cout << "what is the persistent logging database Password ? " << persistentLoggingDatabasePassword << std::endl;
+
+      std::string persistentLoggingDatabaseName = compInfoPT->getCompilationInfo()->getPersistentInfo()->getJITServerPersistentLoggingDatabaseName();
+      std::cout << "what is the persistent logging database Name ? " << persistentLoggingDatabaseName << std::endl;
+
+#ifdef CASSANDRA_LOGGER
+      CassandraLogger* logger = new CassandraLogger(persistentLoggingDatabaseAddress, std::persistentLoggingDatabasePort, persistentLoggingDatabaseName,persistentLoggingDatabaseUsername, persistentLoggingDatabasePassword);
```

Comment:
A new logger is created dynamically, but who destroys it? If you plan on not reusing it, then allocate it on stack rather than with `new`.